.github/workflows/ci.yaml

```diff
@@ -0,0 +1,11 @@
+name: CI
+on:
+  pull_request:
```

Comment:
This will only work when a pull request is opened, but if new changes are introduced GitHub Action will not run. Try this:

```yaml
on: [push, pull_request]
```

In the end, for this PR, you want to have a run of a GitHub Action.